packages/FormElement/src/FormElement.js

```diff
@@ -164,6 +184,15 @@ function FormElement(props) {
                 "aria-readonly": isReadOnly,
               };
 
+              if (child.type.name === componentNameMap.LISTBOX) {
```

Comment:
only the Listbox component is currently making use of the `refLabel` prop. We need to ensure other components such as `Input` do not end up adding this to their generated `input` dom element so i'm being explicit here

Reply:
I would definitely say this should be the responsibility of the `Input` then.

Reply:
Not only `Input` need to take care of it, `TextArea`, `Select`... all the similar components passing `moreProps` directly into DOM element have to take care of it. Is it more complex than doing it like this PR?